sdk/communication/azure-communication-sms/src/test/java/com/azure/communication/sms/SmsTestBase.java

```diff
@@ -3,155 +3,152 @@
-import java.util.stream.Stream;
-
-import reactor.core.publisher.Flux;
+import java.util.Locale;
+import java.util.StringJoiner;
+import java.util.function.Function;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 import reactor.core.publisher.Mono;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.junit.jupiter.api.Assertions.assertTrue;
-import static org.junit.jupiter.api.Assertions.fail;
 
-public class SmsTestBase {
 
-    static final String PROTOCOL = "https://";
-    static final String ENDPOINT = "localhost";
-    static final String PATH = "/sms";
-    static final String ACCESSKEY = "VGhpcyBpcyBhIHRlc3Q="; // Base64 encoded "This is a test"
-    private ObjectMapper mapper;
+public class SmsTestBase extends TestBase {
 
-    public SmsTestBase() {
-        mapper = new ObjectMapper();
-    }
+    protected static final String ENDPOINT = Configuration.getGlobalConfiguration()
+        .get("COMMUNICATION_SERVICE_ENDPOINT", "https://REDACTED.communication.azure.com");
 
-    public SmsClientBuilder getTestSmsClientBuilder(PhoneNumberIdentifier from, List<PhoneNumberIdentifier> to, String body) {
-        SendSmsOptions smsOptions = new SendSmsOptions();
-        smsOptions.setEnableDeliveryReport(false);
-        return getTestSmsClientBuilder(from, to, body, smsOptions, null);
-    }
+    protected static final String ENDPOINT_TOKEN = Configuration.getGlobalConfiguration()
+        .get("COMMUNICATION_TOKEN_ENDPOINT", "https://REDACTED.communication.azure.com");
 
-    public SmsClientBuilder getTestSmsClientBuilder(PhoneNumberIdentifier from, List<PhoneNumberIdentifier> to, String body, SendSmsOptions smsOptions) {
-        return getTestSmsClientBuilder(from, to, body, smsOptions, null);
-    }
+    protected static final String ACCESSKEYRAW = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c";
+    protected static final String ACCESSKEYENCODED = Base64.getEncoder().encodeToString(ACCESSKEYRAW.getBytes());
+    protected static final String ACCESSKEY = Configuration.getGlobalConfiguration()
+        .get("COMMUNICATION_SERVICE_ACCESS_KEY", ACCESSKEYENCODED);
+
+    protected static final String CONNECTION_STRING = Configuration.getGlobalConfiguration()
+        .get("COMMUNICATION_CONNECTION_STRING", "endpoint=https://REDACTED.communication.azure.com/;accesskey=" + ACCESSKEYENCODED);
+
+    protected static final String FROM_PHONE_NUMBER = Configuration.getGlobalConfiguration()
+        .get("FROM_PHONE_NUMBER", "+18335260208");
+
+    protected static final String TO_PHONE_NUMBER = Configuration.getGlobalConfiguration()
+        .get("TO_PHONE_NUMBER", "+18335260208");
+
+    protected static final String FAIL_PHONE_NUMBER = Configuration.getGlobalConfiguration()
+        .get("FAIL_PHONE_NUMBER", "+183352602038");
```

Comment:
We cannot introduce these new variables like "FROM_PHONE_NUMBER", "TO_PHONE_NUMBER" and "FAIL_PHONE_NUMBER ". THe live test pipeline does not have these values and will fail. 
The only phone number that exists is "SMS_SERVICE_PHONE_NUMBER"